lib/nerves_motd.ex

```diff
@@ -12,14 +12,15 @@ defmodule NervesMOTD do
   \e[34m█▌ \e[36m▐█▄▖\e[34m▝▀█▌ \e[36m▐█   \e[39mN  E  R  V  E  S
   \e[34m█▌   \e[36m▝▀█▙▄▖ ▐█
   \e[34m███▌    \e[36m▀▜████\e[0m
-
   """
 
   @typedoc """
   MOTD options
   """
   @type option() :: {:logo, iodata()}
 
+  @typep cell :: {String.t(), iodata()} | {String.t(), iodata(), :green | :red}
```

Comment:
```suggestion
  @typep color() :: :red | :green
  @typep cell() :: {String.t(), iodata()} | {String.t(), iodata(), color()}  
```
It seems like color can be more than just red or green since `IO.ANSI` supports a lot of colors. That can be in a future PR, though.